src/scheduler/job_info.c

```diff
@@ -860,7 +860,7 @@ query_jobs_chunk(th_data_query_jinfo *data)
  * @retval a newly allocated th_data_query_jinfo object
  * @retval NULL for malloc error
  */
-static th_data_query_jinfo *
```

Comment:
Make sure the inline keyword works for all the platforms PBS supports.  It came in the C99 standard.  It works with gcc, but will it work with Windows?

Reply:
We are also removing Windows support for the scheduler with this PR, so Windows shouldn't be an issue. I'll do a build with all other platforms to be sure, thanks.